wormhole-connect/src/store/wallet.ts

```diff
@@ -87,11 +87,12 @@ export const walletSlice = createSlice({
       const { type, error } = payload;
       state[type].error = error;
     },
-    setCurrentAddress: (
+    setAddress: (
       state: WalletState,
       { payload }: PayloadAction<{ type: TransferWallet; address: string }>,
     ) => {
       const { type, address } = payload;
+      state[type].address = address;
```

Comment:
What is the difference between `address` and `currentAddress` in the store?

Reply:
I think it's for cases where the wallet may have different available addresses (I saw some validations using it). However, I only see it set once when connecting and it's never modified again. I didn't want to remove it in case it breaks something, but it may be that the validation is no longer being performed and it's become redundant.